variables.tf

```diff
@@ -16,11 +16,17 @@ variable "tags" {
 ## Azure Automation
 variable "schedules" {
   type = list(object({
-    name      = string
-    frequency = string
-    interval  = number
-    run_time  = string
-    start_vm  = bool
+    name               = string
+    frequency          = string
+    interval           = number
+    run_time           = string
+    start_vm           = bool
+    week_days          = optional(list(string))
+    month_days         = optional(list(number))
+    monthly_occurrence = optional(object({
```

Comment:
Is there any validation we can put on these?
concern is if someone spells Tuesday wrong or puts in a wrong number for days for example.